internal/utils/kruize/kruize_api.go

```diff
@@ -8,11 +8,18 @@ import (
 	"net/http"
 	"strings"
 
+	"github.com/redhatinsights/ros-ocp-backend/internal/config"
+	"github.com/redhatinsights/ros-ocp-backend/internal/logging"
 	"github.com/redhatinsights/ros-ocp-backend/internal/types"
 	"github.com/redhatinsights/ros-ocp-backend/internal/types/kruizePayload"
+	"github.com/redhatinsights/ros-ocp-backend/internal/utils"
+	"github.com/sirupsen/logrus"
 )
 
-func create_kruize_experiments(experiment_name string, k8s_object []map[string]interface{}) ([]string, error) {
+var log *logrus.Logger = logging.GetLogger()
+var cfg *config.Config = config.GetConfig()
```

Comment:
I see you have already added `log` and `cfg` in the `utils` package. Can we use them directly here?

Reply:
No, utils and kruize is a separate package.